project/leonhsi/README.md

```diff
@@ -0,0 +1,47 @@
+# Bounding Volume Hierarchy Trees Generator
+
+###### tags: `Ray Tracing`
+###### links: [link to github repo](https://github.com/leonhsi/BVH_Generator)
+
+## Basic Information
+
+Bounding Volume Hierarchies (BVH) trees are an acceleration sturcture for ray tracing algorithm. The problem of ray tracing is to find the nearest intersection with a given ray and scene. In practice, it has to evaluate billions of such operations as the scene consists of millions of primitives, thus scene primitives are commonly arranged in spatial data structures to accelerate the search. BVH are such data structure that used mostly nowadays.
+
+In this project, I would evaluate each trees with common scenes in computer graphics such as dragon, rabbit, etc. Each of these scenes contains millions of primitives.
```

Comment:
My project will be evaluate with common real world scenes which would contain millions of primitives.